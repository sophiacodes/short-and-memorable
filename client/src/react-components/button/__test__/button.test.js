import React from 'react';
import Button from '../';
import renderer from 'react-test-renderer';

it('renders <Button> correctly', () => {
  const tree = renderer
    .create(<Button value="A button value" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});