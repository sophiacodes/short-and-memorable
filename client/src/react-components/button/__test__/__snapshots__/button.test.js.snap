// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Button> correctly 1`] = `
<input
  className="button"
  type="button"
  value="A button value"
/>
`;
